src/NuGetPackageVerifier/third-party.json

```diff
@@ -37,7 +37,7 @@
     "Newtonsoft.Json": [ "10.0.1" ],
     "Newtonsoft.Json.Bson": [ "1.0.1" ],
     "NuGet.Frameworks": [ "4.0.0" ],
-    "Remotion.Linq": [ "2.1.1" ],
+    "Remotion.Linq": [ "2.2.0-aplha-002" ],
```

Comment:
Pretty sure you mean `alpha`. It sounds like the fixes an important bug for you guys, and that the maintainer says 2.2.0-RTM will be basically identical to the alpha? Just double checking because generally we don't like taking beta/alpha dependencies.